packages/fether-electron/custom.webpack.additions.js

```diff
@@ -0,0 +1,18 @@
+// https://webpack.electron.build/add-ons
+// https://www.npmjs.com/package/webpack-build-notifier
+const path = require('path');
+const WebpackBuildNotifierPlugin = require('webpack-build-notifier');
```

Comment:
I've disabled the Webpack notifications by default now, so you have to run with `NOTIFY=true yarn start` to enable them. I'm also happy to remove since it probably just unnecessarily increases the size of the binary for production too?

Reply:
`NOTIFY=true` sounds good. No it won't increase the size of the binary, since it's not `require()`d by any package inside `fether-electron/src`

Reply:
Edit: Adding the pictures inside `static/` will actually increase the binary's size, since these pictures will be bundled into the binary. Could you move these pictures somewhere else than `static/`? Do we need to specify our own pictures, aren't there any default ones?

Reply:
If you don't provide a picture then it just displays a placeholder icon that looks like a terminal window whether it builds successfully or not.

I've moved the images out of the `static/` folder and fixed the link to the correct .ico file